packages/gatsby-telemetry/src/event-storage.js

```diff
@@ -10,20 +14,52 @@ const isTruthy = require(`./is-truthy`)
  * to continue even when working offline.
  */
 module.exports = class EventStorage {
-  store = new Store()
-  debugEvents = isTruthy(process.env.GATSBY_TELEMETRY_DEBUG)
-  disabled = isTruthy(process.env.GATSBY_TELEMETRY_DISABLED)
+  constructor() {
+    try {
+      this.config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+    } catch (e) {
+      // This should never happen
+      this.config = {
+        get: key => this.config[key],
+        set: (key, value) => (this.config[key] = value),
+        all: this.config,
+        path: os.tmpdir(),
```

Comment:
should we try to create a subfolder here? instead of writing directly into tmpdir?
```suggestion
        path: path.join(os.tmpdir(), 'gatsby'),
```